digital_assets/README.md

```diff
@@ -0,0 +1,26 @@
+# ASWF Digital Assets License 
+
+This folder contains a license for use with for assets intended to be used solely for any of the following purposes:
+
+- education
+- training
+- research
+- software and hardware development
+- performance benchmarking (including publication of benchmark results and permitting reproducibility of the benchmark results by third parties)
+- software and hardware product demonstrations.
+
+Version 1.0 of this license was approved for use by the Academy Software Foundation Governing Board on 2021-07-29. This license is available to used by both ASWF and non-ASWF projects.
+
+## License Attribution
+
```

Comment:
I'd include the following near the top of this section:

When releasing assets under this license, be sure to replace the angle bracketed sections in the header of the template (`<Asset Name>`, `<Year>`, `<Asset Owner>`) with the applicable information for your asset. You may wish to consult with your own legal counsel if you have questions about the specific details to include.